meeting-notes/2019-04-15--libp2p sync.md

```diff
@@ -0,0 +1,130 @@
+# 2019-04-15 libp2p Sync Call
+
+[Issue for this call](https://github.com/libp2p/team-mgmt/issues/16)
+
+**Moderator**: @mgoelzer
+
+**Notetaker**: @jhiesey
+
+**Recording**:  [https://youtu.be/83ihcm-tBQk](https://youtu.be/83ihcm-tBQk)
+
+**Attendees**:
+  - @jhiesey
+  - @marten-seemann
+  - @anacrolix
+  - @mgoelzer
+
+## Agenda
+
+- Weekly updates 
+- (add yours)
+
+## Weekly updates
+
+- @anacrolix: 
+  - https://gist.github.com/anacrolix/95c505eab14ffc283fef932fc5817c54
```

Comment:
@anacrolix – the OC team confirmed you can get the equivalent of gauges by using a measure with lastval view, did that not work for you? https://github.com/census-instrumentation/opencensus-go/issues/1080#issuecomment-481336609